src/ussd_popi_rapidpro.js

```diff
@@ -793,6 +793,193 @@ go.app = function() {
             });
         });
 
+        self.add("state_opt_out", function(name) {
+            return new MenuState(name, {
+                question: $("Do you want to stop getting MomConnect messages?"),
+                error: $(
+                    "Sorry we don't recognise that reply. Please enter the number next to your " +
+                    "answer."
+                ),
+                choices: [
+                    new Choice("state_opt_out_reason", $("Yes")),
+                    new Choice("state_no_optout", $("No"))
+                ]
+            });
+        });
+
+        self.add("state_no_optout", function(name) {
+            return new MenuState(name, {
+                question: $(
+                    "Thanks! MomConnect will continue to send helpful messages and process " +
+                    "your personal info. What would you like to do?"
+                ),
+                error: $(
+                    "Sorry we don't recognise that reply. Please enter the number next to your " +
+                    "answer."
+                ),
+                choices: [
+                    new Choice("state_start", $("Back to main menu")),
+                    new Choice("state_exit", $("Exit"))
+                ]
+            });
+        });
+
+        self.add("state_opt_out_reason", function(name) {
+            return new PaginatedChoiceState(name, {
+                question: $("We'll stop sending msgs. Why do you want to stop your MC msgs?"),
+                error: $("Sorry we don't understand. Please try again."),
+                choices: [
+                    new Choice("miscarriage", $("Miscarriage")),
+                    new Choice("stillbirth", $("Baby was stillborn")),
+                    new Choice("babyloss", $("Baby passed away")),
+                    new Choice("not_useful", $("Msgs aren't helpful")),
```

Comment:
Our reasons match, except in the optout line the `not_useful` is a `not useful`, and everything starts with a capital letter, should I change that on the optout line?

Reply:
Yes, I think I did put a comment on your PR about that. We want our reasons to match https://github.com/praekeltfoundation/ndoh-hub/blob/develop/eventstore/models.py#L35-L42 , so that we can just send it through without having to convert it first